test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -2109,18 +2109,18 @@
         subject (utils/cn "foo")]
     (testing "should not support auto-renewal"
       (is (false? (ca/supports-auto-renewal? (utils/generate-certificate-request keypair subject [] []))))
-      (is (false? (ca/supports-auto-renewal? (utils/generate-certificate-request keypair subject [] [{:oid "1.3.6.1.4.1.34380.1.3.2" :value false}]))))
```

Comment:
Did a find/replace on that magic string with the constant definition in ca.  Thought it was easier to read.